app/configurations/config.okc.js

```diff
@@ -87,10 +87,18 @@ export default configMerger(walttiConfig, {
 
   title: APP_TITLE,
 
-  textLogo: true,
-
+  textLogo: false,
+  logo: 'okc/logo.png',
+  secondaryLogo: 'okc/secondary-logo.png',
```

Comment:
Shall we use a relative path here? Or does the performance penalty not apply here that you have mentioned recently?

Reply:
It does not apply, as these variables are not used by favion-webpack, but appended to a [relative path](https://github.com/HSLdevcom/digitransit-ui/blob/3d6c4a651be87e3d8ab62a824e554ffe6c9080f7/app/component/EmbeddedSearch/EmbeddedSearch.js#L280)